userspace/libsinsp/filter/ast.h

```diff
@@ -162,9 +197,12 @@ struct SINSP_PUBLIC and_expr: expr
 
     std::vector<std::unique_ptr<expr>> children;
 
-    static std::unique_ptr<and_expr> create(std::vector<std::unique_ptr<expr>> &c)
+    static std::unique_ptr<and_expr> create(std::vector<std::unique_ptr<expr>> &c,
+					    const libsinsp::filter::ast::pos_info &pos = s_initial_pos)
```

Comment:
I don't understand why you use the short type `pos_info` a few lines above and the fully qualified type here.
are we out of the `libsinsp::filter::ast::` namespaces?